src/Panel/ChangesDialog.php

```diff
@@ -18,11 +18,9 @@
  */
 class ChangesDialog
 {
-	protected Changes $changes;
-
-	public function __construct()
-	{
-		$this->changes = new Changes();
+	public function __construct(
+		protected Changes $changes = new Changes()
```

Comment:
This works for PHP 8.1+ so I thought it's a neat place to give it a first try.

Reply:
OMG 😍 I don't know how many times I wanted that.